<HTML>
<HEAD>
 <TITLE> BayesPI-BAR</TITLE> 
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-55609707-5', 'auto');
ga('send', 'pageview');
</script>



</HEAD>

<BODY BGCOLOR="FFFFF0"><b></b> 
<h2>
<a href="http://nar.oxfordjournals.org/content/early/2015/07/21/nar.gkv733.abstract">
BayesPI-BAR: A New Biophysical Model for Characterization of Regulatory Sequence Variations
<p></A></b> </h2> 
<br> 

<b><A HREF="mailto:junbai.wang@rr-research.no">Junbai Wang</a> 1</b> and Kirill Batmanov 1 
<br><br> 
<b>
1. Department of Pathology<br>
<br>
Oslo University Hospital - The Norwegian Radium Hospital
<br>
Montebello 0310 Oslo
<br>
Norway

</b> <p> <br>
<b>Abstract
</b><br>
<br><br>
Sequence variation in regulatory DNA regions are known to cause functionally important 
consequences for gene expression. It is a great challenge to identify DNA sequence variations 
which may have essential role in determining phenotype and be linked to disease by analyzing 
massive genome-wide sequencing data. 
<br><br>
Here, a new computational pipeline, Bayesian method 
for Protein-DNA Interaction with Binding Affinity Ranking (BayesPI-BAR), is proposed to 
quantify the effect of sequence variations on protein binding. BayesPI-BAR uses biophysical 
modeling of protein-DNA interaction to predict single nucleotide polymorphisms (SNPs) that 
cause significant changes in the binding affinity of a regulatory region for transcription factors 
(TFs). It includes two new parameters (TF chemical potentials or protein concentrations; and 
direct TF binding targets) that are neglected by the previous methods.
<br><br>
The new method is 
verified at 67 known human regulatory SNPs, where 47 (~70%) have predicted true TFs ranked 
in the top 10. Importantly, the performance of BayesPI-BAR, which uses principal component 
analysis to integrate multiple predictions from various TF chemical potentials, is always better 
than existing programs such as sTRAP and is-rSNP, when evaluated on the same SNPs. 
<br><br>
BayesPI-BAR is a publicly available tool and is able to perform parallelized computation, which helps to 
investigate a large number of TFs and SNPs and to detect disease associated regulatory sequence 
variations, in the sea of genome-wide non-coding regions.

<br><br>
<br>
<br>
</br>
</body>
</html>
